python/ray/serve/_private/deployment_state.py

```diff
@@ -1012,9 +1012,11 @@ def stop(self, graceful: bool = True) -> None:
 
         Should handle the case where the replica is already stopped.
         """
+        state = self._actor_details.state
         logger.info(
-            f"Stopping replica {self.replica_tag} for deployment "
-            f"'{self.deployment_name}' in application '{self.app_name}'.",
+            f"Stopping replica {self.replica_tag} (currently {state}) "
```

Comment:
does this render in a human-readable format?

Reply:
It renders beautifully:

```
2024-02-08 11:41:44,461	INFO deployment_state.py:1916 -- Removing 3 replicas from deployment 'deployment_with_slow_to_start_replicas' in application 'test_app'.
2024-02-08 11:41:44,461	INFO deployment_state.py:1015 -- Stopping replica test_app#deployment_with_slow_to_start_replicas#l7tbrjfb (currently STARTING) for deployment 'deployment_with_slow_to_start_replicas' in application 'test_app'.
2024-02-08 11:41:44,461	INFO deployment_state.py:1015 -- Stopping replica test_app#deployment_with_slow_to_start_replicas#6wjfpi2f (currently RUNNING) for deployment 'deployment_with_slow_to_start_replicas' in application 'test_app'.
2024-02-08 11:41:44,461	INFO deployment_state.py:1015 -- Stopping replica test_app#deployment_with_slow_to_start_replicas#x76wroek (currently RUNNING) for deployment 'deployment_with_slow_to_start_replicas' in application 'test_app'.
```

Reply:
This log is actually from the test run on master, which is obviously failing ^